### ISSUE TITLE
[Bug] The indexer service is missing some data.

### ISSUE THREAD

**Issue by 99Kies** [aptos-labs/aptos-core#7172]
aptos node release: `aptos-node-v1.2.6`
I ran the official indexer service and found that this sync service missed some data during the sync process, some accounts' events information was not synced to the database, I didn't locate the problem for now. (The official indexer-graphql service also has the same problem)
For example, this account.
0x3ce67b2398c742b525606b116bae32d8cfd204c2a19004b3e7fd0ad9951a5e9

aptos explorer:
https://explorer.aptoslabs.com/txn/94139022/events
![image](https://user-images.githubusercontent.com/45918704/225308438-fa98598f-0e0b-4336-b0c0-1beeb7ba3142.png)

Through the explorer, I can observe that this account has events data, but when I check in the indexer database, I can't find any events information in this account. The official graphql service is also can not found.

indexer-graphql-server result:
![b7bced27-b4e7-48c4-ba3e-1b76579193dd](https://user-images.githubusercontent.com/45918704/225308120-8f8a1276-e6d4-4bbd-b825-4469cc2f58cb.jpeg)

**Comment by 99Kies** [comment]
is anyone can help this issue?

**Comment by larry-aptos** [comment]
TLDR: you missed the padding `0` in your query: it should be `0x03ce67b2398c742b525606b116bae32d8cfd204c2a19004b3e7fd0ad9951a5e9`

What's this transaction: Transaction in explorer: https://explorer.aptoslabs.com/txn/94139022/events?network=mainnet

Does this event in indexer: Indexer stores the event: 
![image](https://user-images.githubusercontent.com/112209412/236528617-6e489172-afc0-4810-8d09-4b7b495f2055.png)

`

**Comment by larry-aptos** [comment]
![image](https://user-images.githubusercontent.com/112209412/236532797-ef89fd8f-58c5-4273-85b9-e5e55d5f7bcb.png)

Add GraphQL result.
